src/tests/vos_perf.c

```diff
@@ -666,8 +667,8 @@ ts_yes_or_no(bool value)
 }
 
 const char perf_vos_usage[] = "\n"
-"-f pathname\n"
-"	Full path name of the VOS file.\n\n"
```

Comment:
I remembered that this '-f' option was introduced by me to support DAX device.

The '-f' option could be used to specify a DAX device file, I used this option to evaluate the performance difference of VOS running over "DAX mode ext4" vs. "DAX device". You can see in the vos_pool_destroy(), we still check if the VOS is on a DAX device or a DAX mode fs and perform different destroy operations accordingly.

I think the DAX device support could be removed, and the DAX device checking in vos_pool_destroy() should be removed as well?

Reply:
The problem here was that when Johann tried what was in the docs, it tried to initialize the sys_db in /mnt/daos and failed.   I think we could add the dax support back later if we need it but we don't support it anyway so probably ok to remove.  I'll push another patch.

Reply:
I see. I agree that DAX device support could be removed. Thanks.